securityscanutils/securityscan.go

```diff
@@ -169,13 +170,23 @@ func (r *SecurityScanRepo) RunMarkdownScan(ctx context.Context, client *github.C
 		if err != nil {
 			return eris.Wrapf(err, "error running image scan on image %s", imageWithRepo)
 		}
-		// Create / Update Github issue for the repo if a vulnerability is found
-		// and CreateGithubIssuePerImageVulnerability is set to true
-		if vulnFound && r.Opts.CreateGithubIssuePerImageVulnerability {
-			err = r.CreateUpdateVulnerabilityIssue(ctx, client, imageWithRepo, output)
-			if err != nil {
-				return err
-			}
+		trivyScanMd, err := ioutil.ReadFile(output)
+		if err != nil {
+			return eris.Wrapf(err, "error reading trivy markdown scan file %s to generate github issue", output)
+		}
+
+		if vulnFound {
+			vulnerabilityMd += fmt.Sprintf("# %s\n\n %s\n\n", imageWithRepo, trivyScanMd)
+		}
+
+	}
+	// Create / Update Github issue for the repo if a vulnerability is found
+	// and CreateGithubIssuePerImageVulnerability is set to true
+	if r.Opts.CreateGithubIssuePerImageVulnerability {
```

Comment:
should this be renamed to `CreateGithubIssuePerImage`? or do we still allow the option of creating a separate issue per vulnerability?

Reply:
Ah good catch, will rename.